xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp

```diff
@@ -645,7 +645,7 @@ bool CGUIDialogVideoManagerVersions::GetSimilarMovies(const std::shared_ptr<CFil
 }
 
 bool CGUIDialogVideoManagerVersions::AddSimilarMovieAsVersion(
-    const std::shared_ptr<CFileItem> itemMovie)
+    const std::shared_ptr<CFileItem>& itemMovie)
```

Comment:
Isn't this change also addressed here? https://github.com/xbmc/xbmc/pull/24436#discussion_r1443307905